Task_DEV-2/Checker.cs

```diff
@@ -0,0 +1,157 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace task_DEV_2
+{
+    // Class in which we check the correctness of input srting
+    class Checker
+    {
+        private string expressionOperator = "*-+/";
+
+        // method check input string for correctness 
+        public void CheakInputString(string inputString)
+        {
+            bool error = false;
+            inputString = EditingString(inputString);
+
+            if (CheckRecordNumbersInExponentForm(inputString))
+            {
+                error = true;
+                Console.WriteLine("Check exponention form of numbers!");
+            }
+            if (error == false)
+            {
+                if (BracketsCount(inputString) > 0)
+                {
+                    error = true;
+                    Console.WriteLine("You have more closed brackets!");
+                }
+                if (BracketsCount(inputString) < 0)
+                {
+                    error = true;
+                    Console.WriteLine("You have more open brackets!");
+                }
+            }
+            if (error == false)
+            {
+                if (CheckNeighboringOperator(inputString))
+                {
+                    error = true;
+                    Console.WriteLine("You have 2 or more neighboring operators!");
+                }
+            }
+            if (error)
+            {
+                Console.WriteLine("Error! Check your input data!");
+                Console.ReadKey();
+            }
+            else
+            {
+                Calculator calculator = new Calculator(inputString);
```

Comment:
This is not a proper place to call calculator, CheakInputString is for Check!!!! not to calculate